1209/4646/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SmartPID SPC1000
+owner: SmartPID
+license: GPL V3
+site: http://www.smartpid.com/
+source: https://github.com/arzaman/smartPID
```

Comment:
I don't see any hardware design files here, only schematic PDFs. Can you include hardware source files in your repository, please?